.travis.yml

```diff
@@ -5,13 +5,16 @@ services:
 
 language: java
 before_install:
+  - docker run --name dbfitdb2 -d -p 50000:50000 -v "`pwd`/test_vm/scripts/db2:/var/dbfit/test_vm/scripts/db2" -e DB2INST1_PASSWORD=db2inst1-pwd -e LICENSE=accept ibmcom/db2express-c:latest db2start &
```

Comment:
I'm still wondering if it makes sense to have both `-d` and run it in background in the shell (via `&` suffix).